password_enhancements.module

```diff
@@ -239,36 +286,32 @@ function _password_enhancements_get_roles_from_form_state(array $form, FormState
  */
 function _password_enhancements_get_plugin_settings(array $roles) {
```

Comment:
Can we make this to a class method too?

Reply:
See above: should we create a utility class for these two functions (as static methods) – just for the sake of avoiding the pollution of the global namespace? IDK; that's why their name starts with the module's name.

Reply:
> just for the sake of avoiding the pollution of the global namespace? 

The primary goal is to have unit-testable components in our modules, components that we could replace in our tests if we would like to test edge-cases independent components. This is not possible or really painful with procedural code in tests. See more: https://www.drupal.org/project/drupal/issues/2942529#comment-13133481